expressEngine/model/index.js

```diff
@@ -1,9 +1,9 @@
 import User from './collection/user';
 import Order from './collection/order';
-import Product from './collection/product';
+import ProductModel from './collection/product';
 
 export {
   User,
   Order,
-  Product,
+  ProductModel,
```

Comment:
Look at the naming of other models.
It's User and Order.
So please change ProductModel to Product and also change wherever it is imported as ProductModel.

Reply:
Done! Please have a look.